### PR TITLE
fix: cherry picked all v9.x.x hotfix

### PR DESCRIPTION
**Pull request by ws4charlie**
# Description

Cherry picked v9.x.x hot fixes.

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Include instructions and any relevant details so others can reproduce. 

- [ ] Tested CCTX in localnet
- [ ] Tested in development environment
- [ ] Go unit tests
- [ ] Go integration tests
- [ ] Tested via GitHub Actions 

# Checklist:

- [ ] I have added unit tests that prove my fix feature works

### CODE REVIEW

**Review by lumtis — APPROVED** [review]
LGTM

File: zetaclient/evm_client.go [hunk 1]
```diff
@@ -445,6 +445,13 @@ func (ob *EVMChainClient) IsSendOutTxProcessed(sendHash string, nonce uint64, co
 	return false, false, nil
 }
 
+// The lowest nonce we observe outTx for each chain
+var lowestOutTxNonceToObserve = map[int64]uint64{
+	5:    70000,  // Goerli
+	97:   95000,  // BSC testnet
+	8001: 120000, // Matic
```
Comment (brewmaster012) [inline zetaclient/evm_client.go:452]: matic chainid is 80001

**Review by brewmaster012 — CHANGES_REQUESTED** [review]
minor change request.

**Review by kingpinXD — APPROVED** [review]

File: zetaclient/evm_client.go (hunk 1 above)
Comment (ws4charlie) [reply zetaclient/evm_client.go:452]: Thanks.

**Review by brewmaster012 — APPROVED** [review]

**Pull request merged**